scripts/cc-cli/src/utils/balance.ts

```diff
@@ -1,17 +1,36 @@
 import { BN } from "creditcoin-js";
 
-const MICROUNITS_PER_CTC = new BN("1000000000000000000");
+export const MICROUNITS_PER_CTC = new BN("1000000000000000000");
 
 export function parseCTCString(amount: string): BN {
-  return new BN(amount).mul(MICROUNITS_PER_CTC);
+  const split = amount.split(".");
+  const CTC = split[0];
```

Comment:
The calculations below will probably produce a traceback if:
- I use a different decimal character (it will be counted towards the whole)
- I use non-decimal characters in the input.

Reply:
Do we want to throw an error from here in the case the amount is invalid?